mlflow/sklearn/__init__.py

```diff
@@ -945,6 +1052,58 @@ def patched_fit(original, self, *args, **kwargs):
                     FLAVOR_NAME, class_def, func_name, patched_fit, manage_run=True,
                 )
 
+        for func_name in ["predict", "transform", "predict_proba"]:
+            if hasattr(class_def, func_name):
+                safe_patch(
+                    FLAVOR_NAME, class_def, func_name, patched_predict, manage_run=False,
+                )
+
+    from sklearn import metrics
+    # TODO: patch all metrics API.
+    #   includes all methods ends with _score/_error/_loss in `sklearn.metrics`
+    #   but there're few metric methods ends with other words, like mean_poisson_deviance
+
+    # Note:
+    #  don't need patch model.score() because internally model.score() will call the concrete metric API
+    for metric_method_name in metrics.__all__:
+        # excludes '**Display' object such as `ConfusionMatrixDisplay`
+        if callable(getattr(metrics, metric_method_name)) and not metric_method_name.endswith('Display'):
```

Comment:
Excludes object name ends with `Display` is kind of tricky. Any better approach ?

Reply:
Can we use `inspect.isclass` and exclude classes?

Reply:
We also need to drop `plot_xxx_yyy` functions right?